tasks/main.yml

```diff
@@ -44,14 +50,22 @@
     _docker_os_dist_major_version: 10
   tags: ["install", "configure", "postinstall", "docker_install", "docker_configure", "docker_postinstall"]
 
+- name: Check if /etc/os-release exists
```

Comment:
Did you add this check because that file does not exists in any of the distribution you added support for?

Reply:
Yes, on some distros /etc/os-release doesn't exist.